src/app.jsx

```diff
@@ -32,6 +32,7 @@ import HostVmsList from "./components/vms/hostvmslist.jsx";
 import { StoragePoolList } from "./components/storagePools/storagePoolList.jsx";
```

Comment:
The commit message needs to be cleaned up. There are no "TODO" items in main, nor LRU cache bugs relative to main and the current version of the PR (... hopefully)